### PR TITLE
Updated to add suppressed flags

### PR DESCRIPTION
**Pull request by kmbisset89**
This is a workaround for a multiplatform bug that exists. The IDE flags cannot find the actual even though it exists.

### CODE REVIEW

**Comment by kmbisset89** [comment]
@Alex009 would you mind giving me a review?

**Review by Alex009 — CHANGES_REQUESTED** [review]

File: resources-generator/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```
[the remark this answers is not in the document]
Comment (kmbisset89) [reply resources-generator/gradle/wrapper/gradle-wrapper.properties:1]: Will remove.

File: resources-generator/src/main/kotlin/dev/icerock/gradle/generator/MRGenerator.kt [hunk 2]
```diff
@@ -54,10 +55,19 @@ abstract class MRGenerator(
...
-            .addModifiers(mrSettings.visibility.toModifier())
+            .addModifiers(mrSettings.visibility.toModifier()).also {
+                if (mrSettings.addSuppressAnnotation && getMRClassModifiers().contains(KModifier.EXPECT)) {
+                    it.addAnnotation(
+                        AnnotationSpec.builder(Suppress::class)
+                            .addMember("%S", "NO_ACTUAL_FOR_EXPECT")
+                            .build()
+                    )
+                }
+            }
```
[the remark this answers is not in the document]
Comment (kmbisset89) [reply resources-generator/src/main/kotlin/dev/icerock/gradle/generator/MRGenerator.kt:70]: You are correct. Your code works wonderfully, however if you have a project that does not use commonMain the expect class throws an error saying it cannot find all the actuals even though they exist. In my case I have sourceSets jvmMain, androidMain, and desktopMain where android and desktop dependOn(jvmMain). The proposed change is an optional flag to allow others to compile while multi platform still is in work. I will mark it deprecated when it is no longer needed.

[the remark this answers is not in the document]
Comment (Alex009) [reply resources-generator/src/main/kotlin/dev/icerock/gradle/generator/MRGenerator.kt:70]: please create sample project where your issue can be reproduced. i think we should not use this `NO_ACTUAL_FOR_EXPECT` suppressing, we have some bug maybe

**Review by Alex009 — COMMENTED** [review]
reproducer project required

**Pull request closed without merging**